INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
+cd cactusbot
+pip3 install -r requirements.txt
```

Comment:
Eh, I think that it's fine. Some people still have python 2 as their default, so being explicit is better. Zen of Python, "Explicit is better than implicit"